services/QuillLMS/lib/rule_feedback_history.rb

```diff
@@ -26,6 +26,31 @@ def self.exec_query(conjunction:, activity_id:)
         rules = Comprehension::Rule.find_by_sql(sql)
     end
 
+    def self.feedback_history_to_json(f_h)
+        {
+            response_id: f_h.id,
+            datetime: f_h.updated_at,
+            entry: f_h.entry,
+            highlight: f_h.metadata.class == Hash ? f_h.metadata['highlight'] : '',
+            view_session_url: 'Not yet available',
+            strength: f_h.feedback_history_ratings.order(updated_at: :desc).first&.rating
```

Comment:
Isn't this going to need to be specific to the current user?

Reply:
From the spec: 
`If someone changes a response from weak to strong, it should always show the new option instead.`

I believe the current implementation respects this specification. Note: I implemented this logic expansively  - allowing FeedbackHistory to have multiple ratings - in anticipation of future features (i.e. majority voting). Sorry if that was misleading.